tensorflow/lite/micro/kernels/kernel_util.h

```diff
@@ -31,16 +31,17 @@ namespace micro {
 // The Flexbuffer library is inline heavy, which causes code bloat when
 // custom ops are used. Wrapping with a function is a portable way to avoid
 // this bloat
-const flexbuffers::Map FlexbuffersWrapperGetRootAsMap(const uint8_t* buffer,
-                                                      size_t size);
-
-int32_t FlexbuffersWrapperAsInt32(const flexbuffers::Map& m, const char* key);
-
-bool FlexbuffersWrapperAsBool(const flexbuffers::Map& m, const char* key);
-
-float FlexbuffersWrapperAsFloat(const flexbuffers::Map& m, const char* key);
-
-bool FlexbuffersWrapperIsNull(const flexbuffers::Map& m, const char* key);
+class LiteVector : public flexbuffers::Vector {
+ public:
+  LiteVector(const flexbuffers::Vector& v);
+  int64_t ElementAsInt64(size_t i) const;
+  uint64_t ElementAsUInt64(size_t i) const;
+  int32_t ElementAsInt32(size_t i) const;
+  bool ElementAsBool(size_t i) const;
+};
+
+const LiteVector GetFlexbufferRootAsLiteVector(const uint8_t* buffer,
```

Comment:
nit: document the parameters.

and remove the const from the type that is returned by value: http://go/totw/labs/const#when-should-i-use-const-in-return-types

Actually, it might be more readable to make this a constructor for the LiteVector class:
```cc
class FlexbufferVectorWrapper {

public:
  explicit FlexbufferVectorWrapper(const uint8_t* buffer, size_t size);

...
}

```

And the usage would become:
```
op_data->cycles_max = tflite::FlexbufferVectorWrapper(buffer_t, length).ElementAsInt32(kCyclesMaxIndex);
```